shell/platform/glfw/flutter_glfw.cc

```diff
@@ -330,10 +342,13 @@ static void SetEventLocationFromCursorPosition(
 static void SetEventPhaseFromCursorButtonState(
```

Comment:
Please change the comment on this method, since it's no longer just the primary button. It would also be good to comment that `buttons` needs to be set *before* calling this method. In fact, maybe make `buttons` an argument to make that much more strongly enforced.

Reply:
Updated.